### PR TITLE
Enable bash remediation for SUSE smartcard_pam_enabled

### PR DESCRIPTION
**Pull request by brett060102**
create smartcard_pam_enabled/bash/shared.sh
which includes both sle and ubuntu remediations
use jinja ifs and remove
smartcard_pam_enabled/bash/ubuntu.sh
since that is now in shared.sh

Update test code to support sle.

#### Description:

- Update bash remediation to support sle

#### Rationale:

- ansible not installed by dfault on SLE12/15

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/tests/commented.fail.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
-# platform = multi_platform_ubuntu
+# platform = multi_platform_ubuntu,multi_platform_sle
 # packages = libpam-pkcs11
-
+{{% if product in ["ubuntu1604", "ubuntu1804", "ubuntu2004"] %}}
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/tests/commented.fail.sh:4]:
```suggestion
{{% if 'ubuntu' in product %}}
```

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/bash/shared.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_sle,multi_platform_ubuntu
+{{% if product in ["ubuntu1604", "ubuntu1804", "ubuntu2004"] %}}
```
Comment (Mab879) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/bash/shared.sh:2]:
```suggestion
{{% if 'ubuntu' in product %}}
```

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR. Just a couple of changes requested to make the content more maintainable.

**Comment by brett060102** [comment]
@Mab879 do you want the same {{% if 'ubuntu' in product %}} made to the testing code?

**Comment by Mab879** [comment]
> @Mab879 do you want the same {{% if 'ubuntu' in product %}} made to the testing code?

That would be a good idea

**Comment by brett060102** [comment]
@Mab879 done.

**Comment by brett060102** [comment]
@Mab879 This one is now ready for re-review.

**Comment by brett060102** [comment]
@Mab879 How does this PR look to you now.

**Review by Mab879 — APPROVED** [review]

**Pull request merged**